reference/docs-conceptual/samples/Creating-Get-WinEvent-queries-with-FilterHashtable.md

```diff
@@ -0,0 +1,250 @@
+```Output
+9223372036854775807
+```
+
+For the **Keywords** key, PowerShell uses a number, not a string such as **Security**. **Windows
+Event Viewer** displays the **Keywords** as strings, but they are enumerated values. In the hash
+table, if you use the **Keywords** key with a string value, an error message is displayed.
+
+Open the **Windows Event Viewer** and from the **Actions** pane, click on **Filter current log**.
+The **Keywords** drop-down menu displays the available keywords, as shown in the following
+screenshot:
+
+![Image of Windows Event Viewer keywords.](/media/creating-get-winEvent-queries-with-filterhashtable/keywords.png)
+
+Use the following command to display the `StandardEventKeywords` property names.
+
+```PowerShell
+[System.Diagnostics.Eventing.Reader.StandardEventKeywords] | Get-Member -Static -MemberType Property
+```
+
+```Output
+   TypeName: System.Diagnostics.Eventing.Reader.StandardEventKeywords
+Name             MemberType Definition
+—-             ———- ———-
+AuditFailure     Property   static System.Diagnostics.Eventing.Reader.StandardEventKey…
+AuditSuccess     Property   static System.Diagnostics.Eventing.Reader.StandardEventKey…
+CorrelationHint  Property   static System.Diagnostics.Eventing.Reader.StandardEventKey…
+CorrelationHint2 Property   static System.Diagnostics.Eventing.Reader.StandardEventKey…
+EventLogClassic  Property   static System.Diagnostics.Eventing.Reader.StandardEventKey…
+None             Property   static System.Diagnostics.Eventing.Reader.StandardEventKey…
+ResponseTime     Property   static System.Diagnostics.Eventing.Reader.StandardEventKey…
+Sqm              Property   static System.Diagnostics.Eventing.Reader.StandardEventKey…
+WdiContext       Property   static System.Diagnostics.Eventing.Reader.StandardEventKey…
+WdiDiagnostic    Property   static System.Diagnostics.Eventing.Reader.StandardEventKey…
+```
+
+The enumerated values are documented in the **.NET Framework**. For more information, see
+[StandardEventKeywords Enumeration](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.eventing.reader.standardeventkeywords?redirectedfrom=MSDN&view=netframework-4.7.2).
+
+The **Keywords** names and enumerated values are as follows:
+
+| Name             |  Value            |
+| ---------------- | ------------------|
+| AuditFailure     | 4503599627370496  |
+| AuditSuccess     | 9007199254740992  |
+| CorrelationHint2 | 18014398509481984 |
+| EventLogClassic  | 36028797018963968 |
+| Sqm              | 2251799813685248  |
+| WdiDiagnostic    | 1125899906842624  |
+| WdiContext       | 562949953421312   |
+| ResponseTime     | 281474976710656   |
+| None             | 0                 |
+
+Update the hash table and include the **key/value** pair with the key, **Keywords**, and the
+**EventLogClassic** enumeration value, **36028797018963968**.
+
+```PowerShell
+Get-WinEvent -FilterHashtable @{ LogName='Application'; ProviderName='.NET Runtime'; Keywords=36028797018963968 }
```

Comment:
Reformat all examples with hashtables as follows:

```
Get-WinEvent -FilterHashtable @{ 
   LogName='Application'
   ProviderName='.NET Runtime'
   Keywords=36028797018963968 
}
```

Reply:
Thanks. Added content in the **Hash table key/value pairs** section to explain semicolons and key/value pairs.